### PR TITLE
CB-19562 Specify keytype (RSA or ED25519) during telemetry publisher …

### PR DESCRIPTION
**Pull request by lturcsanyi**
…creation based on FIPS

See detailed description in the commit message.

### CODE REVIEW

File: auth-connector/build.gradle [hunk 1]
```diff
@@ -36,7 +36,7 @@ dependencies {
     exclude group: 'org.mockito'
     exclude group: 'junit', module: 'junit'
   }
-  testImplementation group: 'org.mockito',              name: 'mockito-core',                   version: mockitoVersion
+  testImplementation group: 'org.mockito',              name: 'mockito-inline',                 version: mockitoVersion
```
Comment (lnardai) [inline auth-connector/build.gradle:39]: Why is this needed?

**Review by lnardai — APPROVED** [review]

File: auth-connector/build.gradle (hunk 1 above)
Comment (lturcsanyi) [reply auth-connector/build.gradle:39]: To be able to use Mockito.mockStatic
